src/io/flutter/run/test/FlutterTestEventsConverter.java

```diff
@@ -35,9 +35,9 @@ public FlutterTestEventsConverter(@NotNull String testFrameworkName,
    * Checks if the given group is one created dynamically by flutter_test in the widget_tester
    * widget test wrapper.
    */
-  private static boolean isSyntheticWidgetTestGroup(@NotNull Group group) {
-    return Objects.equals(group.getUrl(), "package:flutter_test/src/widget_tester.dart") &&
-           Objects.equals(group.getName(), SYNTHETIC_WIDGET_GROUP_NAME);
+  private static boolean isSyntheticWidgetTestGroup(@Nullable Item group) {
+    return group instanceof Group && Objects.equals(group.getUrl(), "package:flutter_test/src/widget_tester.dart") &&
+           group.getName().endsWith(SYNTHETIC_WIDGET_GROUP_NAME);
```

Comment:
Well, I *think* so but I'm not sure it's specified so better safe than sorry; will fix.

Thanks!